ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -64,13 +80,53 @@ const styles = StyleSheet.create({
 const LABEL_COL_SIZE_NARROW_LABEL = 1;
 const VALUE_COL_SIZE_NARROW_LABEL = 2;
 
-const LABEL_COL_SIZE_WIDE_LABEL = 1;
+const LABEL_COL_SIZE_WIDE_LABEL = 2;
 const VALUE_COL_SIZE_WIDE_LABEL = 1;
 
 /**
  * Details of transaction
  */
-export class TransactionDetailsScreen extends React.Component<Props, never> {
+export class TransactionDetailsScreen extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      isTransactionCompleted: true
```

Comment:
mmh why the components starts in the _completed_ state? should it be the final state?

Reply:
It should be from the point of view of the user, while the transaction will be executed in the meanwhile. Maybe isTransactionAccepted can be more specific?

Reply:
perhaps `isTransactionStarted`